gp2gp-translator/src/test/java/uk/nhs/adaptors/pss/translator/mapper/diagnosticreport/DiagnosticReportMapperTest.java

```diff
@@ -294,6 +295,156 @@ public void When_DiagnosticReportContainsChildObservationComment_Expect_Observat
+                    </component>
+                    <component typeCode="COMP" contextConductionInd="true">
+                        <NarrativeStatement classCode="OBS" moodCode="EVN">
+                            <id root="3E1A4EA2-661B-4467-8843-6A6B21DEF14F" />
+                            <text>Comments - Aaaargh</text>
+                            <statusCode code="COMPLETE" />
+                            <availabilityTime value="201101111533" />
+                        </NarrativeStatement>
+                    </component>
+                    <component typeCode="COMP" contextConductionInd="true">
+                        <ObservationStatement classCode="OBS" moodCode="EVN">
+                            <id root="58E20D47-F007-47F4-811E-992AE160BECC" />
+                            <code code="7K30.00" codeSystem="2.16.840.1.113883.2.1.6.2" displayName="Total
+                            prosthetic replacement of knee joint using cement">
+                                <translation code="265170009" codeSystem="2.16.840.1.113883.2.1.3.2.4.15" />
+                            </code>
+                            <statusCode code="COMPLETE" />
+                            <effectiveTime>
+                                <center nullFlavor="NI" />
+                            </effectiveTime>
+                            <availabilityTime value="201101111533" />
+                            <pertinentInformation typeCode="PERT">
+                                <sequenceNumber value="+1" />
+                                <pertinentAnnotation classCode="OBS" moodCode="EVN">
+                                    <text>cementing the agreement</text>
+                                </pertinentAnnotation>
+                            </pertinentInformation>
+                        </ObservationStatement>
+                    </component>
+                    <component typeCode="COMP" contextConductionInd="true">
+                        <ObservationStatement classCode="OBS" moodCode="EVN">
+                            <id root="C6D2153A-7EF6-4B1F-8F79-850372BFF67D" />
+                            <code code="7K20.00" codeSystem="2.16.840.1.113883.2.1.6.2" displayName="Total prosthetic
+                            replacement of hip joint using cement">
+                                <translation code="265157000" codeSystem="2.16.840.1.113883.2.1.3.2.4.15" />
+                            </code>
+                            <statusCode code="COMPLETE" />
+                            <effectiveTime>
+                                <center nullFlavor="NI" />
+                            </effectiveTime>
+                            <availabilityTime value="20110104" />
+                        </ObservationStatement>
+                    </component>
+                    <component typeCode="COMP" contextConductionInd="true">
+                        <ObservationStatement classCode="OBS" moodCode="EVN">
```

Comment:
Are these `ObservationStatement`s necessary for the test?

Reply:
I removed it as the test can be run without them.